src/Sulu/Bundle/TestBundle/Testing/AssertHttpStatusCodeTrait.php

```diff
@@ -31,23 +32,32 @@ protected static function assertHttpStatusCode(int $code, Response $response, in
 
         $message = '';
         if ($code !== $httpCode) {
-            $message = $response->getContent();
-
-            if ('null' !== ($json = json_encode(json_decode($message, true), JSON_PRETTY_PRINT))
-                && $json) {
-                $message = explode(PHP_EOL, $json);
+            if ($response->isRedirect()) {
```

Comment:
Why are you doing that? What if I am expecting a redirect?

Reply:
If its the correct status code it will not get to this position because of the if above.
If its the false status code it will end currently with:

```bash
Unexpected redirect to "/sitemaps/pages-1.xml" with status "301".
Failed asserting that 301 is identical to 302.
```

Should be change the text to something like:

```bash
Unexpected "301" status code with redirect to "/sitemaps/pages-1.xml".
Failed asserting that 301 is identical to 302.
```

Or whats your suggestion?

Reply:
And why. Currently the output of a failed assertion looks like this:

```
HTTP status code 301 is not expected 302, showing 50 lines of the response body: <!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8" />
        <meta http-equiv="refresh" content="0;url=/sitemaps/pages-1.xml" />

        <title>Redirecting to /sitemaps/pages-1.xml</title>
    </head>
    <body>
        Redirecting to <a href="/sitemaps/pages-1.xml">/sitemaps/pages-1.xml</a>.
    </body>
</html>
Failed asserting that 301 is identical to 302.
```

The HTML content is for redirects not needed so I want to simplify it and make the `URL` more readable